Tasks/HelmDeployV0/src/helmcommands/helminstall.ts

```diff
@@ -26,17 +26,25 @@ export function addArguments(helmCli: helmcli): void {
     var updatedependency = tl.getBoolInput('updatedependency', false);
     var waitForExecution = tl.getBoolInput('waitForExecution', false);
     var argumentsInput = tl.getInput("arguments", false);
-    var valueFile = tl.getInput("valueFile", false);
+    var valueFilesInput = tl.getInput("valueFile", false);
     var rootFolder = tl.getVariable('System.DefaultWorkingDirectory');
     var enableTls = tl.getBoolInput("enableTls", false);
 
     if (namespace) {
         helmCli.addArgument("--namespace ".concat(namespace));
     }
 
-    if (valueFile && valueFile != rootFolder) {
-        helmCli.addArgument("--values");
-        helmCli.addArgument("\"" + helmutil.resolvePath(valueFile) + "\"");
+    if (valueFilesInput) {
+        let valueFiles = valueFilesInput.split(/[\n,]+/);
+        valueFiles = tl.findMatch(tl.getVariable('System.DefaultWorkingDirectory') || process.cwd(), valueFiles);
```

Comment:
Why the `findMatch` this would allow file paths in patterns to be used. Was that the plan?

Reply:
I used it to add root to unrooted paths. This is what was done with the manifest files in KubernetesManifestV0. I'm not sure if my understanding about this function is correct though.